lib/array_equals.rb

```diff
@@ -1,5 +1,17 @@
 # Determines if the two input arrays have the same count of elements
 # and the same integer values in the same exact order
-def array_equals(array1, array2)
-  raise NotImplementedError
+
+def array_equals(array_a, array_b)
+  if (array_a == [] && array_b == [] || array_a == nil && array_b == nil)
```

Comment:
oh wow! I like how you used a compound conditional to group all the true and false edge cases together. VERY NICE.  ✨✨✨